packages/mui-material-next/src/Menu/Menu.test.tsx

```diff
@@ -594,24 +597,23 @@ describe('<Menu />', () => {
       });
     });
 
-    // TODO v6: Opening on initial mount
-    // it('should open during the initial mount', () => {
-    //   function MenuItem(props) {
-    //     const { autoFocus, children } = props;
-    //     return (
-    //       <div role="menuitem" tabIndex={-1} data-autofocus={autoFocus}>
-    //         {children}
-    //       </div>
-    //     );
-    //   }
-    //   render(
-    //     <ContextMenu anchorEl={document.createElement('div')} open>
-    //       <MenuItem>one</MenuItem>
-    //     </ContextMenu>,
-    //   );
-
-    //   expect(screen.getByRole('menuitem')).to.have.attribute('data-autofocus', 'true');
-    // });
+    it('should open during the initial mount', () => {
+      function MenuItem(props: { children?: React.ReactNode }) {
+        const { children } = props;
+        return (
+          <div role="menuitem" tabIndex={-1}>
+            {children}
+          </div>
+        );
+      }
+      render(
+        <ContextMenu anchorEl={document.createElement('div')} open>
+          <MenuItem>one</MenuItem>
+        </ContextMenu>,
+      );
+
+      expect(screen.getByRole('menuitem')).to.not.equal(null);
```

Comment:
Is the menu item removed from the DOM when the menu is not open? Otherwise, we would have to use another way of testing this

Reply:
> Is the menu item removed from the DOM when the menu is not open?

Yes, we make sure of this in the test "renders its children only when open".